appveyor/Convert-FileUrisToGitHubUris.ps1

```diff
@@ -0,0 +1,120 @@
+$location=$args[0]
+$repoProvider=$env:APPVEYOR_REPO_PROVIDER
+$projectSlug=$env:APPVEYOR_PROJECT_SLUG
+$repoName=$env:APPVEYOR_REPO_NAME
+$repoCommit=$env:APPVEYOR_REPO_COMMIT
+#$repoProvider='gitHub'
+#$projectSlug='sarif-sdk'
+##$projectSlug='sarif-sdk-931m5'
+#$repoName='Rolstenhouse/sarif-sdk'
+#$repoCommit='f82ebd892917bd6157efc04b832a57964ec1f2c6'
+
+$githubBaseUri = ""
+
+if($repoProvider -eq "gitHub")
+{
+  $githubBaseUri = "https://raw.githubusercontent.com/"
+}
+
+function Rebase-Uri($originalURI)
+{
+  $caseSensitiveUri = Get-CaseSensitivePath($originalURI.SubString($originalURI.IndexOf("c:/")))
+  return $githubBaseUri + $repoName+"/"+$repoCommit+"/"+$caseSensitiveUri.SubString($caseSensitiveUri.IndexOf($projectSlug)+$projectSlug.Length+1)
+}
+
+function Get-CaseSensitivePath($pathName)
+{
+  $pathExists = Test-Path($pathName)
+  if (-Not $pathExists)
+  {
+      return $pathName
+  }
+  $directoryInfo = New-Object IO.DirectoryInfo($pathName)
+
+  if ($directoryInfo.Parent -ne $null)
+  {
+      $parentPath = Get-CaseSensitivePath($directoryInfo.Parent.FullName)
+      $childPath = $directoryInfo.Parent.GetFileSystemInfos($directoryInfo.Name)[0].Name
+      return(Join-Path $parentPath $childpath -resolv)
+  }
+  else
+  {
+      return $directoryInfo.Name
```

Comment:
Later, everything before the repository name will be rewritten anyway. I think dealing with the edge case of having the repository name as the root with incorrect casing, will not occur